src/lib/strategies/AbstractKeyEventStrategy.js

```diff
@@ -1080,12 +1092,17 @@ class AbstractKeyEventStrategy {
       combinationMatch.events[eventBitmapIndex];
 
     if (!combinationHasHandlerForEventType) {
+      /**
+       * If the combination does not have any actions bound to the key event we are
+       * currently processing, we skip checking if it matches the current keys being
+       * pressed.
+       */
       return false;
     }
 
     let keyCompletesCombination = false;
 
-    const combinationMatchesKeysPressed = !Object.keys(combinationMatch.keyDictionary).some((candidateKeyName) => {
+    const combinationMatchesKeysPressed = Object.keys(combinationMatch.keyDictionary).every((candidateKeyName) => {
```

Comment:
🐥Switch from a negated `.some` function to an `.every` function for understandability.